<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>755</width>
    <height>608</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>MS Outlook</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QFrame#Main {
	background-color: qlineargradient(spread:pad, x1:0.472045, y1:0, x2:1.156818, y2:0.744, stop:0 rgba(255, 0, 0, 255), stop:1 rgba(2, 32, 43, 251));
	border-radius:10px;
}

QLabel#size{
	color : yellow;
	background-color: rgba(0, 0, 0,80);

}

QFrame#Preview {
	background-color: white;
	border-radius:10px;
}

QPushButton#Closed{
	background-color: rgba(0, 0,0, 0);
	color:rgba(120,115,255,255);
	border-radius:5px;
}
QPushButton#Closed:hover{
	color:rgba(250,223,11,255);
}
QPushButton#Closed:pressed{
	padding-left:1px;
	padding-top:1px;
	background-color:rgba(150,123,111,255);
}
QPushButton#fButton,#iButton,#tButton{
	background-color: rgba(0, 0,0, 0);
	color:rgba(120,215,255,255);
	
	border-radius:5px;
}
QPushButton#fButton:hover,#iButton:hover,#tButton:hover{
	color:rgba(250,223,11,255);
}
QPushButton#fButton:pressed,#iButton:pressed,#tButton:pressed{
	padding-left:1px;
	padding-top:1px;
	background-color:rgba(150,123,111,255);
}
QPushButton#btn_generatePic,#btn_downloadPic,#btn_setWallpaper{
	border-radius:10px;
	background-color: rgba(190, 155, 255,255);
	color:rgba(0,0,0,255);
	font: 8pt;
}
QPushButton#btn_generatePic:hover,#btn_downloadPic:hover,#btn_setWallpaper:hover{
	color:rgba(255,255,200,255);
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(187, 100, 229, 255), stop:1 rgba(255, 155, 255, 255));
	border-radius:10px;
	}
QPushButton#btn_generatePic:pressed,#btn_downloadPic:pressed,#btn_setWallpaper:pressed{
	border-radius:10px;
	padding-left:2px;
	padding-top:2px;
	background-color:rgba(150,123,111,255);
}

QLabel#Title{
	
	color: rgba(0,9,0,200);
	font:18pt;
}
QLabel#status{
color:yellow;
}


QPushButton#Expand,#Minimize{
	background-color: rgba(0, 0,0, 0);
	color:rgba(120,115,255,255);
	border-radius:5px;
}
QPushButton#Expand:hover,#Minimize:hover{
	color:rgba(250,223,11,255);
}
QPushButton#Expand:pressed,#Minimize:pressed{
	padding-left:1px;
	padding-top:1px;
	background-color:rgba(150,123,111,255);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="Main">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>15</y>
      <width>190</width>
      <height>370</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="Title">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>30</y>
       <width>171</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Megrim</family>
       <pointsize>18</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="text">
      <string>WALLPAPER DOWNLOADER</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="status">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>340</y>
       <width>170</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Palatino Linotype</family>
       <pointsize>7</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="Closed">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>10</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QLabel" name="followme">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>170</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
color:black;</string>
     </property>
     <property name="text">
      <string>Follow me on </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>290</y>
       <width>171</width>
       <height>35</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="fButton">
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Social Media Circled</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>E</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="tButton">
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Social Media Circled</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>D</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="iButton">
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Social Media Circled</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>C</string>
        </property>
       </widget>
      </item>
     </layout>
     <zorder>tButton</zorder>
     <zorder>fButton</zorder>
     <zorder>iButton</zorder>
    </widget>
    <widget class="QPushButton" name="btn_generatePic">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>171</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Generate Picture</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Expand">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>138</y>
       <width>40</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS UI Gothic</family>
       <pointsize>32</pointsize>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>‚ùØ</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="Preview_ImageOut">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>151</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sabon</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(85, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_setWallpaper">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>171</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Set Wallpaper</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Minimize">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>138</y>
       <width>40</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS UI Gothic</family>
       <pointsize>32</pointsize>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>‚ùÆ</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="Preview">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>0</width>
      <height>330</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="Title_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>281</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Megrim</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="text">
      <string>WALLPAPER PREVIEW</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>270</y>
       <width>261</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="Download_text">
        <property name="text">
         <string> Downloads :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="No_Downloads">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="Lenght_text">
        <property name="text">
         <string>Lenght :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="Width">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="times">
        <property name="text">
         <string>x</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="Height">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="btn_downloadPic">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>300</y>
       <width>261</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Palatino Linotype</family>
       <pointsize>8</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Set As Wallpaper</string>
     </property>
    </widget>
    <widget class="QLabel" name="pic_View">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>261</width>
       <height>211</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="size">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>230</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Rockwell Extra Bold</family>
      </font>
     </property>
     <property name="text">
      <string>   File Size : </string>
     </property>
    </widget>
   </widget>
   <zorder>Preview</zorder>
   <zorder>Main</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
